recipe/meta.yaml

```diff
@@ -1,29 +1,28 @@
 {% set name = "radical.pilot" %}
-{% set version = "1.49.2" %}
-{% set sha256 = "f49f40426df2326b1c724b4a4bcd34c2fe07b60c96040e55c0a129a3d13646cc" %}
+{% set version = "1.52.1" %}
```

Comment:
I'm skipping over some other recent versions.  Let me know if there's a reason to build those in between versions, too.